docs/conf.py

```diff
@@ -1,237 +1,14 @@
-"""Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
-
-   Permission is hereby granted, free of charge, to any person obtaining a copy
-   of this software and associated documentation files (the "Software"), to deal
-   in the Software without restriction, including without limitation the rights
-   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell cop-
-   ies of the Software, and to permit persons to whom the Software is furnished
-   to do so, subject to the following conditions:
-
-   The above copyright notice and this permission notice shall be included in all
-   copies or substantial portions of the Software.
-
-   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IM-
-   PLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-   FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-   COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-   IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNE-
-   CTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-"""
-
-# -*- coding: utf-8 -*-
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Did we lose a copyright statement in this change?

Reply:
Yes, I decided to remove it because this repository has a file dedicated to the license statements: https://github.com/ROCmSoftwarePlatform/rocRAND/blob/develop/LICENSE.txt

Same for rocm-docs-core, which this code references: https://github.com/RadeonOpenCompute/rocm-docs-core/blob/main/LICENSE.txt

Reply:
We are handling copyrights for the website centrally. No need to per repository copyright content for the website.